pom.xml

```diff
@@ -123,30 +123,30 @@
   </pluginRepositories>
 
   <properties>
-    <amqp-10-starter.version>2.3.3</amqp-10-starter.version>
-    <dekorate.version>1.0.3</dekorate.version>
-    <infinispan-starter.version>2.3.3.Final</infinispan-starter.version>
+    <amqp-10-starter.version>2.3.6</amqp-10-starter.version>
+    <dekorate.version>2.1.3</dekorate.version>
+    <infinispan-starter.version>2.3.6.Final</infinispan-starter.version>
     <!-- https://github.com/snowdrop/narayana-spring-boot -->
     <narayana.version>5.10.6.Final</narayana.version>
-    <narayana-starter.version>2.5.0</narayana-starter.version>
+    <narayana-starter.version>2.5.3</narayana-starter.version>
     <!-- https://github.com/opentracing-contrib/java-spring-jaeger -->
     <opentracing-spring-jaeger-web-starter.version>3.2.2</opentracing-spring-jaeger-web-starter.version>
-    <resteasy.version>3.14.0.Final</resteasy.version>
+    <resteasy.version>3.6.4.Final</resteasy.version>
```

Comment:
In [this jira](https://issues.redhat.com/browse/EAPSUP-502) it's said that resteasy-jaxrs shoud be 3.15.1, why a different version of the one indicated [here](https://issues.redhat.com/browse/RESTEASY-2873)?

Reply:
You're right, I used the starter version instead of the actual resteasy version.